pom.xml

```diff
@@ -306,5 +319,49 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jreleaser</groupId>
+            <artifactId>jreleaser-maven-plugin</artifactId>
+            <version>0.6.0</version>
+            <configuration>
+              <jreleaser>
+                <project>
+                  <snapshot>
+                    <label>{{ projectVersionNumber }}-early-access</label>
+                  </snapshot>
+                </project>
+                <distributions>
+                  <kcctl>
+                    <name>kcctl</name>
+                    <type>NATIVE_IMAGE</type>
+                    <artifacts>
+                      <artifact>
+                        <path>{{artifactsDir}}/{{distributionName}}-{{projectVersion}}-linux-x86_64.zip</path>
+                        <transform>artifacts/{{distributionName}}-{{projectEffectiveVersion}}-linux-x86_64.zip</transform>
+                        <platform>linux-x86_64</platform>
+                      </artifact>
```

Comment:
Isn't this missing the TAR.GZ flavour?

Reply:
Yes. Are we producing one?

Reply:
According to `src/main/assembly/assembly.xml` we only produce ZIP files for distributions.

Reply:
Hum, that's really weird. You are right, but I could swear I configured it recently to produce both TAR.GZ and ZIP.

Reply:
Wans't that the case for kcetcd?

Reply:
Ah yes, of course. Ok, so at least I'm not dreaming :) Could you update the PR so we create and publish both formats?

Reply:
OK, so you want kcctl to publish 6 artifacts in total?